### DISCUSSION TITLE
Password hashing inside of edge runtime?

### DISCUSSION THREAD

**Discussion by igmtink** [honojs/hono#2530] [Q&A]
how to hash a password inside of edge runtime? I tried to use `bcrypt` but It's not available node in edge runtime, also I'm using vercel

**Comment by yudai-nkt** [comment]
This repository has shown up my GitHub feed recently and you might want to give it a try: https://github.com/bruceharrison1984/bcrypt-edge

**Reply by yusukebe** [reply]
I also use it.

**Comment by VamsiPunnaReddy** [comment]
Will it work on cloudflare workers

I tried it and it is showing error
